python/lsst/ap/association/utils.py

```diff
@@ -141,7 +142,7 @@ def convertTableToSdmSchema(apdbSchema, sourceTable, tableName):
 
     Parameters
     ----------
-    apdbSchema : `lsst.dax.apdb.apdbSchema.ApdbSchema`
+    apdbSchema : 'dict' of `lsst.dax.apdb.apdbSchema.ApdbSchema`
```

Comment:
This should be `lsst.dax.apdb.schema_model.Table` instead of `lsst.dax.apdb.apdbSchema.ApdbSchema`. Same on the line 128 above.